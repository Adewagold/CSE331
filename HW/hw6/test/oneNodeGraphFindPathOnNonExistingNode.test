# Tests find path on one_node_graph

# Load oneNodeGraph.tsv
LoadGraph one_node_graph oneNodeGraph.tsv

# Find minimum-cost path with non-existing node
FindPath one_node_graph onlyNode a
FindPath one_node_graph b onlyNode 