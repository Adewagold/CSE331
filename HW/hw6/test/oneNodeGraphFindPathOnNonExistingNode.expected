# Tests find path on one_node_graph

# Load oneNodeGraph.tsv
loaded graph one_node_graph

# Find minimum-cost path with non-existing node
unknown character a
unknown character b